keras_cv/layers/preprocessing/mosaic_test.py

```diff
@@ -86,16 +86,6 @@ def test_image_input_only(self):
         ):
             _ = layer(xs)
 
-    def test_single_image_input(self):
```

Comment:
(This test case is also still relevant since we're keeping the batch-checking logic)